Terminal.Gui/Application/Application.Mouse.cs

```diff
@@ -62,7 +55,7 @@ public static void UngrabMouse ()
         }
 
 #if DEBUG_IDISPOSABLE
-        ObjectDisposedException.ThrowIf (MouseGrabView.WasDisposed, typeof (View));
```

Comment:
typeof was better

Reply:
If you would like the consumer to benefit from better specificity of the most-derived type, rather than always calling it just a `View`, you can just make that method generic:

```csharp
public static void GrabMouse<T> (T? view) where T : View
```

and then the only change necessary should be that the typeof uses turn into `typeof(T)` instead of `typeof(View)`.